lib/chef/audit/default_attributes.rb

```diff
@@ -0,0 +1,123 @@
+
+        # If enabled, a cache is built for all backend calls. This should only be
+        # disabled if you are expecting unique results from the same backend call.
+        "inspec_backend_cache" => true,
+
+        # controls where inspec scan reports are sent
+        # possible values: 'chef-server-automate', 'chef-automate', 'json-file'
+        # notes: 'chef-automate' requires inspec version 0.27.1 or greater
+        # deprecated: 'chef-visibility' is replaced with 'chef-automate'
+        # deprecated: 'chef-compliance' is replaced with 'chef-automate'
+        # deprecated: 'chef-server-visibility' is replaced with 'chef-server-automate'
+        "reporter" => "json-file",
+
+        # controls where inspec profiles are fetched from, Chef Automate or via Chef Server
+        # possible values: nil, 'chef-server', 'chef-automate'
+        "fetcher" => nil,
+
+        # allow for connections to HTTPS endpoints using self-signed ssl certificates
+        "insecure" => nil,
+
+        # Optional for 'chef-server-automate' reporter
+        # defaults to Chef Server org if not defined
+        "owner" => nil,
+
+        # raise exception if Automate API endpoint is unreachable
+        # while fetching profiles or posting a report
+        "raise_if_unreachable" => true,
+
+        # fail converge if downloaded profile is not present
+        # https://github.com/chef-cookbooks/audit/issues/166
+        "fail_if_not_present" => false,
+
+        "interval" => {
+          # control how often inspec scans are run, if not on every node converge
+          # notes: false value will result in running inspec scan every converge
+          "enabled" => false,
+
+          # controls how often inspec scans are run (in minutes)
+          # notes: only used if interval is enabled above
+          "time" => 1440,
+        },
+
+        # controls verbosity of inspec runner
+        "quiet" => true,
+
+        # Chef Inspec Compliance profiles to be used for scan of node
+        # See README.md for details
+        "profiles" => {},
+
+        # Attributes used to run the given profiles
+        "attributes" => {},
+
+        # Set this to false if you don't want ['audit']['attributes'] to be saved in the node object and stored in Chef Server or Automate. Useful if you are passing sensitive data to the inspec profile via the attributes.
+        "attributes_save" => true,
+
+        # If enabled, a hash of the Chef "node" object will be sent to InSpec in an attribute
+        # named `chef_node`
+        "chef_node_attribute_enabled" => false,
+
+        # Set this to the path of a YAML waiver file you wish to apply
+        # See https://www.inspec.io/docs/reference/waivers/
+        "waiver_file" => nil,
+
+        "json_file" => {
+          # The location of the json-file output:
+          # <chef_cache_path>/cookbooks/audit/inspec-<timestamp>.json
+          # TODO: ^^ comment is wrong
+          # TODO: Does this path work?
```

Comment:
TODOs

Reply:
I'm not sure what the right thing to do is here. audit-cookbook uses a path relative to the attributes file: https://github.com/chef-cookbooks/audit/blob/master/attributes/default.rb#L87-L90

The value I have as the default currently uses a path relative to this file, which will likely not work if this is installed as a gem. Should this be something like `File.join(Chef::Config.cache_path, "inspec_reports")`?